modules/web/js/ballerina/env/enum.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import _ from 'lodash';
+import BallerinaEnvFactory from './ballerina-env-factory';
+
+/**
+ * @class Enum
+ * @augments
+ * @param {Object} args data to create the enum
```

Comment:
args are being duplicated in the constructor too

Reply:
Fixed with 533f6ba5b5a478b16da7f64f7acdafaf47712d8d